keps/1136-provisioning-request-support/README.md

```diff
@@ -170,6 +172,59 @@ type ProvisioningRequestConfigSpec struct {
 	// +listType=set
 	// +kubebuilder:validation:MaxItems=100
 	ManagedResources []corev1.ResourceName `json:"managedResources,omitempty"`
+
+	// retryStrategy defines strategy for retrying ProvisioningRequest
+	//
+	// +optional
+	RetryStrategy *ProvisioningRequestRetryStrategy `json:retryStrategy, omitempty`
+}
+
+type ProvisioningRequestRetryStrategy struct {
```

Comment:
What is the differences between requeuingStrategy and this?
As I can see this KEP, I'm not sure the relationship between those, and what if we specify both.

Reply:
This field contains also `Requeue` field which defines whether during retry we should requeue Workload or just recreate ProvisioningRequest (as it's done now). I introduced the same fields as in RequeuingStrategy but didn't want to use the type RequeuingStrategy itself to decouple this and WaitForPodsReady

Reply:
This is resolved by offline discussion as I left a comment here: https://github.com/kubernetes-sigs/kueue/pull/3278#discussion_r1809050147